examples/persistent_subscriptions.rs

```diff
@@ -92,3 +92,108 @@ async fn delete_persistent_subscription(client: &Client) -> eventstore::Result<(
+    let info = client
+        .get_persistent_subscription_info("$all", "subscription-group", &Default::default())
+        .await?;
+
+    println!(
+        "GroupName: {}, EventStreamId: {}, Status: {:?}",
+        info.group_name, info.event_stream_id, info.status
+    );
+    // #endregion get-persistent-subscription-to-all-info
+    Ok(())
+}
+
+async fn replay_parked_to_stream(client: &Client) -> eventstore::Result<()> {
+    // #region replay-parked-of-persistent-subscription-to-stream
+    let options = ReplayParkedMessagesOptions::default().stop_at(10);
+    client
+        .replay_parked_messages("test-stream", "subscription-group", &options)
+        .await?;
+    // #endregion replay-parked-of-persistent-subscription-to-stream
+    Ok(())
+}
+
+async fn replay_parked_to_all(client: &Client) -> eventstore::Result<()> {
+    // #region replay-parked-of-persistent-subscription-to-all
+    let options = ReplayParkedMessagesOptions::default().stop_at(10);
+    client
+        .replay_parked_messages("$all", "subscription-group", &options)
+        .await?;
+    // #endregion replay-parked-of-persistent-subscription-to-all
+    Ok(())
+}
+
+async fn list_persistent_subscription_to_stream(client: &Client) -> eventstore::Result<()> {
+    // #region list-persistent-subscriptions-to-stream
+    let subscriptions = client
+        .list_persistent_subscriptions_for_stream("test-stream", &Default::default())
+        .await?;
+
+    for s in subscriptions {
+        println!(
+            "GroupName: {}, EventStreamId: {}, Status: {:?}",
+            s.group_name, s.event_stream_id, s.status
+        );
+    }
+    // #endregion list-persistent-subscriptions-to-stream
+    Ok(())
+}
+
+async fn list_persistent_subscription_to_all(client: &Client) -> eventstore::Result<()> {
+    // #region list-persistent-subscriptions-to-all
+    let subscriptions = client
+        .list_persistent_subscriptions_for_stream("$all", &Default::default())
+        .await?;
+
+    for s in subscriptions {
+        println!(
+            "GroupName: {}, EventStreamId: {}, Status: {:?}",
+            s.group_name, s.event_stream_id, s.status
+        );
+    }
+    // #endregion list-persistent-subscriptions-to-all
+    Ok(())
+}
+
+async fn list_all_persistent_subscription(client: &Client) -> eventstore::Result<()> {
+    // #region list-persistent-subscriptions
+    let subscriptions = client
+        .list_all_persistent_subscriptions(&Default::default())
+        .await?;
+
+    for s in subscriptions {
+        println!(
+            "GroupName: {}, EventStreamId: {}, Status: {:?}",
+            s.group_name, s.event_stream_id, s.status
+        );
+    }
+    // #endregion list-persistent-subscriptions
+    Ok(())
+}
+
+async fn restart_persistent_subscription_subsystem(client: &Client) -> eventstore::Result<()> {
+    // #region restart-persistent-subscription-subsystem
+    client
+        .restart_persistent_subscription_subsystem(&Default::default())
```

Comment:
**CONCERN:** Test for restart is missing.

Reply:
This is not testable. It returns nothing and there is no clue it actually performed well.

Reply:
Understood. Still, it's making the call to the database, so we could at least check if the result is ok (so no panic here etc. like here https://doc.rust-lang.org/std/result/#results-must-be-used)

Reply:
Done